pages/services/include/security-configure-transport-encryption.tmpl

```diff
@@ -26,7 +26,7 @@ Install the DC/OS {{ model.techName }} service including the following options i
         "service_account_secret": "<full path of service secret>",
         "security": {
             "transport_encryption": {
-                "enabled": true{{ model.security.plaintext }}
```

Comment:
Some services support plaintext (which is why this template was there). 

What's the issue that was occurring?

Reply:
Ideally we'd be able to just correct the template such that the formatting works out right.

@nickbp @elezar have either of you had better luck with special symbols in a template?

Reply:
Slathering it with triple brackets may help: `{{{foo}}}`